### ISSUE TITLE
Add CI and local modes

### ISSUE THREAD

**Issue by tombruijn** [tombruijn/lintje#3]
Not always are all rules as applicable. Locally I want to create fixup and squash commits all the time and it shouldn't complain, or instead warn rather than error. On the CI environment it should fail on these types of commits.

```
# Run in local mode (useful for Git hooks)
lintje --local
lintje -l

# Run in CI mode (useful for in automated builds)
lintje --ci
```

**Comment by tombruijn** [comment]
This is blocked on https://github.com/tombruijn/lintje/issues/24 where I'm working on different levels of severity for rules.
